presto-main/src/test/java/com/facebook/presto/execution/MockQueryExecution.java

```
@@ -182,9 +182,8 @@ public Throwable getFailureCause()
     }
 
     @Override
-    public ListenableFuture<QueryOutputInfo> getOutputInfo()
+    public void addOutputInfoListener(Consumer<QueryOutputInfo> listener)
     {
-        return SettableFuture.create();
```

Add a `// no-op` comment, or should this throw `UnsupportedOperationException`?